inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -147,6 +148,7 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
     decomp->add_matcher<ngraph::pass::DropoutWithRandomUniformReplacer>();
     decomp->set_name("ngraph::pass::CommonDecompositions");
 
+    manager.register_pass<ngraph::pass::ConvertPriorBox8To0>();  // not plugins implemented priorbox8
```

Comment:
Please move it to the section below where other downgrade passes are located.

Reply:
updated in commit 5cf6467